Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -839,12 +688,11 @@ static bool CheckReadPreconditions()
   return true;
 }
 
-// Iff false is returned, ScheduleEvent must be used to finish executing the command
-static bool ExecuteReadCommand(u64 dvd_offset, u32 output_address, u32 dvd_length,
-                               u32 output_length, const DiscIO::Partition& partition,
-                               ReplyType reply_type, DIInterruptType* interrupt_type)
+// If false is returned, ScheduleEvent must be used to finish executing the command
```

Comment:
FYI, `Iff` wasn't a typo. But if you prefer it this way, we can do it this way.

Reply:
Oh okay, I've never seen that abbreviation before. I'll change it back.